src/Microsoft.DotNet.Build.Tasks.Feed/ConfigureInputFeed.cs

```diff
@@ -0,0 +1,57 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Build.Utilities;
+using System.IO;
+using Microsoft.Build.Framework;
+using System;
+
+namespace Microsoft.DotNet.Build.Tasks.Feed
+{
+    public class ConfigureInputFeed : Task
+    {
+        [Required]
+        public ITaskItem[] EnableFeeds { get; set; }
+
+        public ITaskItem[] DisableFeeds { get; set; }
+
+        public string RepoRoot { get; set; }
+
+        public override bool Execute()
+        {
+            GenerateNugetConfig(EnableFeeds, DisableFeeds, RepoRoot, Log);
+            return !Log.HasLoggedErrors;
+        }
+
+        public void GenerateNugetConfig(ITaskItem[] EnableFeeds, ITaskItem[] DisableFeeds, string RepoRoot, TaskLoggingHelper Log)
+        {
+            if (string.IsNullOrWhiteSpace(RepoRoot)) {
+                RepoRoot = Directory.GetCurrentDirectory();
+            }
+            string nugetConfigLocation = Path.Combine(RepoRoot, "NuGet.config");
+
+            Log.LogMessage(MessageImportance.High, $"START Writing NuGet.config to {nugetConfigLocation}...");
+            string nugetConfigBody = $"<?xml version=\"1.0\" encoding=\"utf-8\"?>{Environment.NewLine}<configuration>{Environment.NewLine}";
+            nugetConfigBody += $"<!--Don't use any higher level config files.{Environment.NewLine}Our builds need to be isolated from user/ machine state-->{Environment.NewLine}";
+            nugetConfigBody += $"<fallbackPackageFolders>{Environment.NewLine}<clear />{Environment.NewLine}</fallbackPackageFolders>{Environment.NewLine}<packageSources>{Environment.NewLine}<clear />{Environment.NewLine}";
+            for(int i=0;i<EnableFeeds.Length;i++)
+            {
+                nugetConfigBody += $"<add key=\"inputFeed{i}\" value=\"{ EnableFeeds[i].ItemSpec }\" />{Environment.NewLine}";
+            }
+            nugetConfigBody += $"</packageSources>{Environment.NewLine}";
+            nugetConfigBody += $"<disabledPackageSources>{Environment.NewLine}";
+            for (int i = 0; i < DisableFeeds.Length; i++)
+            {
+                nugetConfigBody += $"<add key=\"disableFeed{i}\" value=\"{ DisableFeeds[i].ItemSpec }\" value=\"true\" />{Environment.NewLine}";
```

Comment:
What cases do we care about DisabledFeeds? 

Unless I'm miss-understanding the spec https://docs.microsoft.com/en-us/nuget/schema/nuget-config-file#disabledpackagesources I think the feed needs to be listed in the package source list and the key be listed here. You cannot have two values. 

I personally would just removed the DisabledFeed stuff unless we have a scenario for it right now.